Samples/Windowing/cs-winui/TitleBar.xaml

```diff
@@ -16,9 +16,44 @@
-                <Button x:Name="TitlebarCustomBtn" Click="TitlebarCustomBtn_Click">Toggle custom TitleBar</Button>
-                <Button x:Name="ResetTitlebarBtn" Click="ResetTitlebarBtn_Click">Reset Titlebar</Button>
+                <Grid>
+                    <Grid.RowDefinitions>
+                        <RowDefinition Height="Auto"></RowDefinition>
+                        <RowDefinition Height="Auto"></RowDefinition>
+                        <RowDefinition Height="Auto"></RowDefinition>
+                        <RowDefinition Height="Auto"></RowDefinition>
+                        <RowDefinition Height="Auto"></RowDefinition>
+                        <RowDefinition Height="Auto"></RowDefinition>
+                        <RowDefinition Height="Auto"></RowDefinition>
+                        <RowDefinition Height="Auto"></RowDefinition>
+                        <RowDefinition Height="Auto"></RowDefinition>
+                    </Grid.RowDefinitions>
+
+                    <TextBlock Grid.Row="0" Margin="0,10,10,0" TextWrapping="Wrap"><Bold>Apply color customizations </Bold> to the defaut title bar </TextBlock>
+                    <StackPanel Grid.Row="1" Spacing="10" Margin="0,0,0,10"  Orientation="Horizontal" VerticalAlignment="Center">
+                        <Button x:Name="TitlebarBrandingBtn" Click="TitlebarBrandingBtn_Click">Toggle title bar color customizations</Button> 
+                    </StackPanel>
+
+
+                    <TextBlock Grid.Row="2" Margin="0,10,10,0" TextWrapping="Wrap"><Bold>Implement a fully custom title bar</Bold> with AppWindowTitleBar.ExtendsContentIntoTitleBar = True </TextBlock>
+                    <StackPanel Grid.Row="3" Spacing="10" Margin="0,0,0,10"  Orientation="Horizontal" VerticalAlignment="Center">
+                        <Button x:Name="TitlebarCustomBtn" Click="TitlebarCustomBtn_Click">Toggle fully custom title bar</Button>
+                    </StackPanel>
+
+                    <TextBlock Grid.Row="4" Margin="0,10,10,0" TextWrapping="Wrap">
+                        <Bold>AppWindow Tall titlebar support:</Bold>
+                        You can choose between standard or tall title bar heights with AppWindowTitleBar.PreferredHeightOption API when AppWindowTitleBar.ExtendsContentIntoTitleBar = True to create more room for interactive content such as a searchbox or person picture. <Italic>Toggle the fully custom titlebar first to enable tall titlebar support</Italic>.
+                    </TextBlock>
+                    <StackPanel Grid.Row="5" Spacing ="10" Orientation="Horizontal" VerticalAlignment="Center">
+                        <Button x:Name="StandardHeightBtn" IsEnabled="False" Click="StandardHeightBtn_Click" MinWidth="100">Standard height title bar</Button>
+                        <Button x:Name="TallHeightBtn" IsEnabled="False" Click="TallHeightBtn_Click" MinWidth="100">Taller title bar</Button>
```

Comment:
Any particular reason these are two buttons instead of a toggle switch?

Reply:
Sam and I thought it might be better with two. They do not break each other. It's inconsistent with the other buttons but perhaps that's a good thing.

Reply:
Ok. Consider it more of a "code style/nit" from my side than something that need to be fixed/changed if you're both happy with this approach.

I just thought that since this is a toggle kind of an experience it mapped more directly to a XAML toggle switch (you can't apply standard height to an already standard height titlebar and vice versa) and you'd not end up with "dead buttons" (buttons that don't do anything when you click them). Plus, by using a toggle switch you can use the state of the switch to check which experience to apply instead of creating a global variable tracking this state.